tests/test-todd-coxeter.cpp

```diff
@@ -2747,10 +2746,12 @@ namespace libsemigroups {
     REQUIRE(tc.number_of_classes() == 36'412);
   }
 
+  // This test seems to segfault, including on debug mode, and it causes JDE's
+  // terminal to crash making it tought to figure out what's going wrong.
   LIBSEMIGROUPS_TEST_CASE("ToddCoxeter",
                           "068",
                           "Walker 5",
-                          "[todd-coxeter][extreme]") {
+                          "[todd-coxeter][fail]") {
```

Comment:
This test (and several others that I have marked with the same comment) consistently crashed my terminal. It's not clear if this is a me issue, but I have marked them with fail to be safe. If anyone else is able to reproduce, or not, I'd be interested to know!